### PR TITLE
[Quality] Remove deprecation import

### PR DESCRIPTION
**Pull request by yatho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -892,6 +892,7 @@ to `AUTO_DEFERRED` and manually call `MatomoInitializerService.initializeTracker
         return () =>
           http.get('/my-config').pipe(tap(config => matomoInitializer.initializeTracker(config)));
       },
+      deps: [HttpClient],
```
Comment (EmmanuelRoux) [inline README.md]: `deps` is useless since `inject` is used

**Comment by EmmanuelRoux** [comment]
Hi @yatho thanks for the PR, there's indeed a pending rewrite of the README

In the meantime your PR is useful 👍

**Review by yatho — COMMENTED** [review]
[Quality] Remove deps useless in this case.

**Review by EmmanuelRoux — APPROVED** [review]

**Comment by yatho** [comment]
This lib is great, thank you to share it.

Actually I write with standalone writing.

I didn't know for deps, I have deleted the line.

**Comment by EmmanuelRoux** [comment]
Thanks for the feedback

Actually I plan to completely rewrite the docs now that standalone providers are preferred, when I have some time

**Pull request merged**